src/helpers/network/truncateMiddleIfNeeded.ts

```diff
@@ -0,0 +1,7 @@
+export default function (str: string, maxLength: number) {
+  const isOddNum = maxLength % 2 === 1 ? 1 : 2
```

Comment:
I think this is `isOddLength`, btw we're using same function in many projects, probably worth extracting it into the package

Reply:
I'll add task for this one